x-pack/plugins/saved_objects_tagging/public/management/components/action_bar.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState, useCallback, useMemo, FC } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiPopover,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiText,
+  EuiContextMenu,
+  EuiContextMenuPanelItemDescriptor,
+  EuiHorizontalRule,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { TagBulkAction } from '../types';
+
+import './_action_bar.scss';
+
+export interface ActionBarProps {
+  actions: TagBulkAction[];
+  totalCount: number;
+  selectedCount: number;
+  onActionSelected: (action: TagBulkAction) => void;
+}
+
+const actionToMenuItem = (
+  action: TagBulkAction,
+  onActionSelected: (action: TagBulkAction) => void,
+  closePopover: () => void
+): EuiContextMenuPanelItemDescriptor => {
+  return {
+    name: action.label,
+    icon: action.icon,
+    onClick: () => {
+      closePopover();
+      onActionSelected(action);
+    },
+    'data-test-subj': `actionBar-button-${action.id}`,
+  };
+};
+
+export const ActionBar: FC<ActionBarProps> = ({
+  actions,
+  onActionSelected,
+  selectedCount,
+  totalCount,
+}) => {
+  const [isPopoverOpened, setPopOverOpened] = useState(false);
+
+  const closePopover = useCallback(() => {
+    setPopOverOpened(false);
+  }, [setPopOverOpened]);
+
+  const togglePopover = useCallback(() => {
+    setPopOverOpened((opened) => !opened);
+  }, [setPopOverOpened]);
+
+  const contextMenuPanels = useMemo(() => {
+    return [
+      {
+        id: 0,
+        items: actions.map((action) => actionToMenuItem(action, onActionSelected, closePopover)),
+      },
+    ];
+  }, [actions, onActionSelected, closePopover]);
+
+  return (
+    <div className="tagMgt__actionBar">
+      <EuiFlexGroup justifyContent="flexStart" alignItems="baseline" gutterSize="m">
+        <EuiFlexItem grow={false}>
+          <EuiText size="xs" color="subdued">
+            <FormattedMessage
+              id="xpack.savedObjectsTagging.management.actionBar.totalTagsLabel"
+              defaultMessage="{count, plural, one {1 tag} other {# tags}}"
+              values={{
+                count: totalCount,
+              }}
+            />
+          </EuiText>
+        </EuiFlexItem>
+        {selectedCount > 0 && (
+          <EuiFlexItem grow={false}>
```

Comment:
Assuming this condition is true, the design includes a vertical border/divider that appears to separate the current item count from the bulk actions. Would it be possible to add?

![image](https://user-images.githubusercontent.com/3884767/98692123-c5e41080-233c-11eb-96c2-a59f7d4e8dda.png)

Reply:
@cchaos does EUI provide such vertical spacers?

Reply:
No it does not. I don't think there's been any other usages of such a divider either.

Reply:
Done in https://github.com/elastic/kibana/pull/82816/commits/aea10e6dc5ff34bf59c92d50ae18213909fd7389